internal/devbox/util.go

```diff
@@ -32,14 +35,66 @@ func (d *Devbox) addDevboxUtilityPackage(ctx context.Context, pkgName string) er
 		return err
 	}
 
-	return nixprofile.ProfileInstall(ctx, &nixprofile.ProfileInstallArgs{
```

Comment:
If `nixprofile.ProfileInstall` and `nixprofile.ProfileInstallArgs` are no longer used you should remove.

Reply:
I was a little confused about having both nixprofile and nix.profile. Is nix.profile the one we should keep using? Can we completely remove nixprofile?